pfcpiface/main.go

```diff
@@ -56,6 +65,14 @@ type CPIfaceInfo struct {
 	FQDNHost string `json:"hostname"`
 }
 
+// P4rtcInfo : P4 runtime interface settings
+type P4rtcInfo struct {
+	AccessIP    string `json:"access_ip"`
+	P4rtcServer string `json:"p4rtc_server"`
+	P4rtcPort   string `json:"p4rtc_port"`
+	UEIP        string `json:"ue_ip_pool"`
+}
+
```

Comment:
Can we move structs and parsing of bess/p4 files to their own files?

Reply:
done